### ISSUE TITLE
java.lang.AbstractMethodError

### ISSUE THREAD

**Issue by iandow** [biboudis/jmh-profilers#8]
I'm getting this error after running a JMH benchmark with -prof jfr:

```
# Processing profiler results: jfr Exception in thread "main" java.lang.AbstractMethodError:      
profilers.FlightRecordingProfiler.afterTrial(Lorg/openjdk/jmh/results/BenchmarkResult;JLjava
/io/File;Ljava/io/File;)Ljava/util/Collection;
    at org.openjdk.jmh.runner.Runner.runSeparate(Runner.java:668)
at org.openjdk.jmh.runner.Runner.runBenchmarks(Runner.java:536)
at org.openjdk.jmh.runner.Runner.internalRun(Runner.java:308)
at org.openjdk.jmh.runner.Runner.run(Runner.java:203)
at org.openjdk.jmh.Main.main(Main.java:69)
```

The full listing of my JMH command and results are here:
http://pastebin.com/Qwig21vh

**Comment by jzillmann** [comment]
Having the same with jmh-1.17.4!

**Comment by biboudis** [comment]
Haven't used it in a long time and the project wasn't properly modularized! @jzillmann, @iandow. 
I'll try to have a look.

**Comment by leogomes** [comment]
@biboudis https://github.com/biboudis/jmh-profilers/pull/10 should fix the issue. Let me know if you have any comments.

Cheers,
Leo.

**Comment by biboudis** [comment]
Fixed by #10

**Comment by jzillmann** [comment]
Thanks, works nicely now!
